atom/browser/atom_browser_client.cc

```diff
@@ -90,14 +98,22 @@ bool AtomBrowserClient::ShouldCreateNewSiteInstance(
     content::BrowserContext* browser_context,
     content::SiteInstance* current_instance,
     const GURL& url) {
-
+  if (g_suppress_renderer_process_restart) {
```

Comment:
Can I only allow `about:blank` as synchronously-opened window for now?

https://github.com/electron/electron/blob/3f7b3c4bd7d3fe2babadc96c0e4b019526d4100f/lib/browser/guest-window-manager.js#L113-L148
In the current implementation, if the URL opened with `window.open` is not `about:blank`, `createGuest` causes `OverrideSiteInstanceForNavigation`, and that's why I did workaround with `g_reuse_renderer_process_for_new_window` variable and prevented renderer process spawn in `ShouldCreateNewSiteInstance`.

If `about:blank` is only allowed as synchronously-opened window, I can remove `g_reuse_renderer_process_for_new_window` completely.